webapp/app.py

```diff
@@ -17,6 +29,15 @@
 login_manager.init_app(app)
 socketio.init_app(app)
 
+# Enable gzip compression of static assets
+compress.init_app(app)
+
+# Enable indefinite caching of static assets based on hash values
+cache_buster.init_app(app)
```

Comment:
We need to be more cautious about what code actually lives in this file. Generally speaking, if the flask extension is invoked using an `init_app(app)`, then all extension realated should go into a separate file from which we import an object named `cache_buster` into this file

Reply:
caching and compression has been moved to separate file